flutter/lib/desktop/pages/install_page.dart

```diff
@@ -120,6 +114,8 @@ class _InstallPageBodyState extends State<_InstallPageBody>
 
   @override
   Widget build(BuildContext context) {
+    final buttonStyle = Theme.of(context).outlinedButtonTheme.style;
```

Comment:
This won't change button style?

Reply:
Please paste the screenshot before and after the change.

Reply:
this can change button style but in MyTheme is:
```dart
OutlinedButton.styleFrom(
        backgroundColor: grayBg,
        foregroundColor: Colors.black87,
        shape: RoundedRectangleBorder(
          borderRadius: BorderRadius.circular(8.0),
        ),
      )
```
need OutlinedButton style is:
```dart
OutlinedButton.styleFrom(
    textStyle: TextStyle(fontSize: 14, fontWeight: FontWeight.normal),
    padding: EdgeInsets.symmetric(vertical: 15, horizontal: 12),
  )
```
change MyTheme is Ok？